app/components/Article/styles.scss

```diff
@@ -4,6 +4,7 @@
 .article {
   @include newspaper;
   transition: opacity $transition-slow;
+  width: 600px;
```

Comment:
In my experience limiting the width of paragraphs is easier for reading. When limiting the width the open space on each side of text act as guides and I find myself losing my place in the text less frequently. Also if you compare the wide text format to the thin text view there is considerably less white space in the thin view that your eye has to traverse. Less movement means less time and less strain for reading.